test/libraries/Compounding.sol

```diff
@@ -43,7 +43,15 @@ interface IEulerToken {
   function underlyingAsset() external view returns(address);
 }
 
+interface ILidoToken {
+  /// @notice Returns amount of stETH for a given amount of wstETH
+  function getStETHByWstETH(uint256) external view returns (uint256);
+  /// @dev NOTE there is no underlying asset here as Compounding stores the Lido stETH address directly
+}
+
 library Compounding {
+  // TODO can we dynamically fetch this? Other options?
+  address constant LIDO_STETH = '0xae7ab96520de3a18e5e111b5eaab095312d7fe84';
```

Comment:
This seems to be something like `aaveAddr` in Swivel.sol. It's a fixed address so it won't change.

Also we can get the stETH address with the [`stETH` method](https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#readContract#F9) on wstETH.